app/models/sample.rb

```diff
@@ -459,6 +459,10 @@ def self.my_data(user)
     where("project_id in (?)", project_ids)
   end
 
+  def role_name
```

Comment:
I think you meant to put this on user.rb right? since you call sample.user.role_name above

Reply:
Yes, that was my intention. Thank you for catching this one.

Reply:
I'm adding a test around that method to prevent regressions

Reply:
done